spec/commands/v2/put_link_set_spec.rb

```diff
@@ -42,4 +42,15 @@
       topics: [topic_uuid],
     )
   end
+
+  it "creates related LinkedItems" do
+    described_class.call(link_params)
+
+    first_link = LinkedItem.first
+    second_link = LinkedItem.last
```

Comment:
These are relying on an ordering that's not guaranteed - there's nothing that guarantees that the first linked item will be the one you're interested in.  It would be better to make these queries specific to the item you're looking for.

Additionally, you're looking up `second_link`, but not then asserting against it below.